nilearn/decomposition/canica.py

```diff
@@ -206,8 +206,8 @@ def _unmix_components(self, components):
         # Note: fastICA is very unstable, hence we use 64bit on it
         results = Parallel(n_jobs=self.n_jobs, verbose=self.verbose)(
             delayed(self._cache(fastica, func_memory_level=2))
-            (components.astype(np.float64), whiten=True, fun='cube',
-             random_state=seed)
+            (components.astype(np.float64), whiten='arbitrary-variance',
```

Comment:
The whitening change also maters for getting rid of the error ?

Reply:
No but using `True` as the value is deprecated and is equivalent to this argument. However, if it's confusing I'll remove the commit handling the warnings since it's the case that these changes are unrelated to the PR and can be handled in another.

Reply:
Ok I got rid of the 2 changes in the first commit. The tests should still pass.